xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -847,17 +847,23 @@ void CPVRClients::ConnectionStateChange(CPVRClient* client,
       return;
   }
 
+  // Use addon-supplied eventlog title if present
+  std::string strTitle;
+  if (!strEventTitle.empty())
+    strTitle = strEventTitle;
+  else
+    strTitle = client->GetFriendlyName();
+
   // Use addon-supplied message, if present
   std::string strMsg;
   if (!strMessage.empty())
     strMsg = strMessage;
   else
-    strMsg = g_localizeStrings.Get(iMsg);
+    strMsg = client->GetFriendlyName();
```

Comment:
This change looks definitely wrong. `iMsg` contains the id of the various connection status strings, like "connecting", "connection lost" etc...

Reply:
It is definitely wrong and corrected but I still need to retest the state changes.